manubot/pandoc/tests/test_cite_filter/input.md

```diff
@@ -0,0 +1,45 @@
+---
+# yaml_metadata_block with pandoc metadata
+citekey-aliases:
+  tag:meta-review: url:https://greenelab.github.io/meta-review/
```

Comment:
Is this `tag:meta-review:` syntax temporary to be backwards compatible?  Or is this how you envision using tags/aliases in the future?

Reply:
Backwards compatible syntax that is required until we implement #192. After we implement #192 then you will no longer need to prefix with `tag:`. But for now it's needed.